frontend/packages/operator-lifecycle-manager/integration-tests/views/operator.view.ts

```diff
@@ -3,6 +3,8 @@ import { Descriptor } from '../../src/components/descriptors/types';
 
 export const rowForOperator = (displayName: string) =>
   $(`[data-test-operator-row="${displayName}"]`);
+export const operatorNameLink = (displayName: string) =>
+  $(`[data-test-operator-row="${displayName}"]`).$('.co-clusterserviceversion-logo__name');
```

Comment:
I'm unsure why this is necessary. The data-test attribute should be on the link already.

Reply:
Somehow the original code wouldn't work with Firefox, but picking a little deeper element inside would. I'm open to any better ideas to try.